src/shared/utils/get-custom-tags.ts

```diff
@@ -1,7 +1,11 @@
-export const getCustomTags = async () => {
+export const getCustomTags = async (isStaged=false) => {
 
     const hname = window.location.hostname
-    const url = `https://d3wuj95q2emo9.cloudfront.net/${Buffer.from(hname, 'utf-8').toString('base64')}.js`
+    var url = `https://test-custom-tags.cnna.io/${Buffer.from(hname, 'utf-8').toString('base64')}.js`
+    if(isStaged){
+        url = `https://d3wuj95q2emo9.cloudfront.net/${Buffer.from(hname, 'utf-8').toString('base64')}.js`
+    }
```

Comment:
This can just be an `env` variable.... instead of evaluating `isStaged=false` so we don't have to add an additional parameter to the tracker...

Something like

```ts
const url = `${process.env.FRICTIONLESS_CLOUDFRONT_URL}/${Buffer.from(hname, 'utf-8').toString('base64')}.js`
```

Then we'd set up different urls for the staging branch and master branch.

I.E


For staging
```env
FRICTIONLESS_CLOUDFRONT_URL=https://test-custom-tags.cnna.io
```

For master
```env
FRICTIONLESS_CLOUDFRONT_URL=https://d3wuj95q2emo9.cloudfront.net
```